shell/exec.go

```diff
@@ -102,8 +103,24 @@ func (s *MenderShell) pipeStdout() {
 		raw := make([]byte, 255)
```

Comment:
This might be optimized by the compiler, but can we move the buffer allocation outside the loop?